core/src/main/java/org/elasticsearch/search/aggregations/pipeline/bucketmetrics/BucketMetricsPipelineAggregator.java

```diff
@@ -49,6 +49,7 @@
     protected GapPolicy gapPolicy;
 
     public BucketMetricsPipelineAggregator() {
+        // NOCOMMIT remove me
```

Comment:
AH! Yeah, this is a leftover. I think I can just remove the method. If not I'll convert.